src/bumprace.c

```diff
@@ -51,6 +57,14 @@
        player user[MAX_PLAYER_NUM]; int pl=0,playernum=2;
     //general setting
        int final=1,fullscreen=1,pageflip=0,sound=1,precision=10,fps=0,particle=1,mode,dofadeout=1,levelset=0;
+
+#ifdef NET
```

Comment:
Any reason against always compiling with network support? That's plenty of `#ifdef`s.

Reply:
I thought it would be nice to have the option to choose to compile this into the game and it made it clear where I have worked. I can remove them if you like